meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor/update-balena-supervisor

```diff
@@ -140,12 +140,21 @@ else
     tag=$UPDATER_SUPERVISOR_TAG
 fi
 
+setSupervisorConfig() {
+    svconfigdir=$(ls /etc/ | grep "supervisor" | head -n 1)
```

Comment:
@acostach sorry I am late to this review. This line issues a warning in shellcheck and it does not look robust enough as it relies on the first hit being the `balena|resin-supervisor` directory. Maybe something like:
```
svconfigdir=$(basename "$(ls -d /etc/*-supervisor)")
```
Is more robust.